uniform-tensors/tensors/D-extend.rkt

```diff
@@ -190,7 +190,8 @@
            (size-out (size-of s-out))
            (v-out (new-vec size-out 0.0)))
       (for ([i-out (in-range 0 size-out stride-out)]
-            [i0 (in-range off0 (+ off0 size0) stride0)])
+            #;[i0 (in-range off0 (+ off0 size0) stride0)])
+        (define i0 (+ off0 (* (/ i-out stride-out) stride0)))
```

Comment:
No nested defines in our coding conventions :) Use a let. 

Same two comments on line 216 and 217